src/main/java/essenmakanan/parser/IngredientParser.java

```diff
@@ -59,6 +59,11 @@ public static Ingredient parseIngredient(String inputDetail)
         assert (ingredientDetails.length == 3) : "Ingredient details should have 3 parts";
 
         String ingredientName = ingredientDetails[0].strip();
+        if (ingredientName.isEmpty()) {
+            // ingredient name cannot be empty
```

Comment:
Does this comment value add to your clear if statement?

Reply:
ok will remove